src/CreateAnnotationDialog.js

```diff
@@ -177,16 +124,18 @@ class CreateAnnotationDialog extends EventEmitter {
      *
      * @return {void}
      */
-    updatePosition() {
+    updatePosition = () => {
         if (this.isMobile) {
             return;
         }
 
+        const popoverEl = findElement(this.annotatedElement, '.ba-popover', this.renderAnnotationPopover);
+
         // Plus 1 pixel for caret
-        this.containerEl.style.left = `${this.position.x - 1 - this.containerEl.clientWidth / 2}px`;
```

Comment:
using a CSS calc might be better than doing it in JS

Reply:
I can't do that if I'm using the coordinates which are determined via the JS right?